### PR TITLE
Updated necessary values to go with new example  version.

### PR DESCRIPTION
**Pull request by catgualter**
Was trying out the new example version this morning and came to the following issues but it was just a simple fix as reference to  #11 and of updating the values to have the intended results back as previous.

Updated the necessary values to:
- Fixed bug in clientApplication when user tries to sign in but does not redirect/popup but instead goes to the sign in page again.
- Fixed bug in resourceServer where user does not go to "Login completed..."  when clicking the redirect_ui received from authorisation grant result.

Thank you again for creating this package. Hope to contribute more.

### CODE REVIEW

File: examples/clientApplication/client/index.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ $(document).ready(function() {
         var setCount = 0;
         setCount += prePopulateValues(e, 'configure-login-service-dialog-clientId', 'clientApplication');
         setCount += prePopulateValues(e, 'configure-login-service-dialog-secret', '12345');
-        setCount += prePopulateValues(e, 'configure-login-service-dialog-baseUrl', 'http://localhost:3100');
```
Comment (vangorra) [inline examples/clientApplication/client/index.js:6]: While this appears counter intuitive, the examples/clientApplication example is intended to work with the examples/resourceServer which runs on localhost:3100.

File: examples/resourceServer/client/index.html [hunk 2]
```diff
@@ -24,7 +24,7 @@
     {{#unless isUrlParamsValid }}
         <p>The URL params are not set in order to authorize an oauth2 token.</p>
         <p>The following link will simulate arriving to this page from another site that wishes to authenticate with oauth2.</p>
-        <a href="/?client_id=resourceServerClient&redirect_uri=http://localhost:3200/_oauth/MeteorOAuth2Server&response_type=code">Simulate proper link.</a>
+        <a href="/?client_id=clientApplication&redirect_uri=http://localhost:3100/_oauth/MeteorOAuth2Server&response_type=code">Simulate proper link.</a>
```
Comment (vangorra) [inline examples/resourceServer/client/index.html:27]: The client_id is a good catch and was a mistake. There is a pending pull request that fixes this. Refer to the previous comment about the port number.

File: examples/resourceServer/server/index.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ function initializeClients() {
         {
             clientId: 'clientApplication', // unique identifier.
             active: true, // whether this client is enabled or not.
-            redirectUri: 'http://localhost:3200/_oauth/MeteorOAuth2Server', // provided by the client.
+            redirectUri: 'http://localhost:3100/_oauth/MeteorOAuth2Server', // provided by the client.
```
Comment (vangorra) [inline examples/resourceServer/server/index.js:10]: The client_id is a good catch and was a mistake. There is a pending pull request that fixes this. Refer to the previous comment about the port number.

File: examples/clientApplication/client/index.js (hunk 1 above)
Comment (catgualter) [inline examples/clientApplication/client/index.js:6]: I must have overlooked it. I was using port 3100 for the clientApplication and port 3200 for the resourceServer.

**Comment by vangorra** [comment]
Fixed in #10. Thank you for the PR.

**Pull request closed without merging**